src/utils/mtev_log.c

```diff
@@ -539,12 +538,12 @@ static void *
 asynch_logio_writer(void *vls) {
   mtev_log_stream_t ls = vls;
   asynch_log_ctx *actx = ls->op_ctx;
-  int gen;
-  gen = mtev_atomic_inc32(&actx->gen);
+  uint32_t gen;
+  gen = ck_pr_faa_32(&actx->gen, 1) + 1;
```

Comment:
Are you sure you want to faa( gen, 1) also add one? +1? Seems

Reply:
mtev_atomic_inc32 did a cas-based add (of 1) and returned the new value.  So, yes?